react/store/createReturnRequest/components/ConfirmAndSubmit.tsx

```diff
@@ -55,8 +58,9 @@ export const ConfirmAndSubmit = ({ onPageChange, items }: Props) => {
         throw new Error('Error creating return request')
       }
 
-      if (data?.createReturnRequest?.returnRequestId) {
-        setRequestId(data.createReturnRequest.returnRequestId)
+      setShowSubmittedStatus(true)
+      if (errors) {
+        setConfirmationStatus('error')
```

Comment:
The error handler here is a little bit confusing and not catching all possibilities. We can review that together.